posthog/async_migrations/migrations/0002_events_sample_by.py

```diff
@@ -104,6 +104,17 @@ def operations(self):
 
         last_partition_op = [generate_insert_into_op(self._partitions[-1] if len(self._partitions) > 0 else 0)]
 
+        def optimize_table_fn():
+            default_timeout = getattr(config, "ASYNC_MIGRATIONS_DEFAULT_TIMEOUT")
+            sync_execute(
```

Comment:
we're missing query ID here too + any other future changes there, so instead add additional_setting or sth like that to `execute_op_clickhouse` and use that fn here too.